### PR TITLE
Make "OPTS UTF8 ON" optional

### LINKED ISSUE #172: Add Dial option to disable UTF8
**Describe the bug**
The FTP server one of my client's ISP is using doesn't recognize the UTF8 option and refuses to continue from there. I had to comment the line in the library to get it to work. 

**To Reproduce**
```
Connect to ftpcluster.loopia.se and enable UTF8.
```

### PR DESCRIPTION
**Pull request by crazy-max**
Closes #172

### CODE REVIEW

File: ftp.go [hunk 1]
```diff
@@ -564,6 +531,36 @@ func (c *ServerConn) List(path string) (entries []*Entry, err error) {
 	return
 }
 
+// SetUTF8 issues an "OPTS UTF8 ON" command.
```
Comment (jlaffaye) [inline ftp.go:534]: Can you rename the function in-place to no mess up with git history ?

Comment (crazy-max) [reply ftp.go]: Ofc, done.

**Comment by crazy-max** [comment]
@jlaffaye, Do you need anything else to merge this PR?

**Comment by crazy-max** [comment]
@jlaffaye Up

**Comment by crazy-max** [comment]
@jlaffaye Up

**Comment by crazy-max** [comment]
Hi @jlaffaye, is there anything missing to merge this PR? Thanks.

**Comment by jlaffaye** [comment]
My main concern is that it changes the default behaviour of the client.
So instead of making it opt-in, maybe we should make it opt-out ?

**Comment by crazy-max** [comment]
@jlaffaye 

> So instead of making it opt-in, maybe we should make it opt-out ?

Sounds good

**Comment by crazy-max** [comment]
@jlaffaye PR has been updated.

**Comment by jlaffaye** [comment]
Thanks!

**Pull request merged**